enhancements/20240605-enhanced-maintenance-mode.md

```diff
@@ -0,0 +1,154 @@
+2. Make sure a second VM without the `harvesterhci.io/maintain-mode-strategy` label is running on the same node. Migrate it to the node on which the other VM is running if necessary.
+3. Note the node on which the VMs are running.
+4. Go to the `Hosts` page and select the `Enable Maintenance Mode` action in the actions dropdown menu of the node your VMs are running on.
+5. After some time the `Cordon` state should be displayed for the host we are putting into maintenance mode.
+6. Now go to the `Virtual Machines` page. The VM with the `harvesterhci.io/maintain-mode-strategy` label should show the `Stopping` label. The other VM should show the `Migrate` status badge.
+7. Choose the `Edit YAML` action menu on the powered off VM and make sure the `harvesterhci.io/maintain-mode-strategy-node-name: <NODENAME>` is in the annotations.
+8. Go back to the `Hosts`page. Wait until the `Maintenance`status badge is shown.
+9. On the `Virtual Machines` page, the VM that was shut down should be restarted and running on a different node than before. Note, this VM was NOT migrated; it was shutdown and restarted.
+10. On the `Hosts` page, disable the maintenance mode via the action menu of the node. After some time the `Maintenance Mode` status badge should be gone away.
+11. Go to the `Virtual Machines` page. The VMs should still run and the node on which they run should not have changed again.
+
+In the logs of the `harvester-xxx-xxx` pods you should find something like this:
+```
+time="2024-03-19T13:00:30Z" level=info msg="attempting to place node in maintenance mode" node_name=harvester-node-1
+time="2024-03-19T13:00:30Z" level=info msg="force stopping VM" namespace=default virtualmachine_name=test01
+time="2024-03-19T13:00:31Z" level=info msg="migration of pod owned by VM test01 is skipped because of the label harvesterhci.io/maintain-mode-strategy" namespace=default pod_name=virt-launcher-test01-rsvwf
+time="2024-03-19T13:01:19Z" level=info msg="restarting the VM that was temporary shut down for maintenance mode" namespace=default virtualmachine_name=test01
+```
+
+#### Case 3
+1. Add the `harvesterhci.io/maintain-mode-strategy` label to a VM of your choice. Press `Edit YAML` and add `harvesterhci.io/maintain-mode-strategy: Shutdown` to `metadata.labels`. Save the settings.
+2. Make sure a second VM without the `harvesterhci.io/maintain-mode-strategy` label is running on the same node. Migrate it to the node on which the other VM is running if necessary.
+3. Note the node on which the VMs are running.
+4. Go to the `Hosts` page and select the `Enable Maintenance Mode` action in the actions dropdown menu of the node your VMs are running on.
+5. After some time the `Cordon` state should be displayed for the host we are putting into maintenance mode.
+6. Now go to the `Virtual Machines` page. The VM with the `harvesterhci.io/maintain-mode-strategy` label should show the `Stopping` label. The other VM should show the `Migrate` status badge.
```

Comment:
show the `Stopping` label  
or
show the `Stopping` status badge?

Reply:
I changed `label` => `status badge` because this should be much clearer to the user.